sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/implementation/BatchClientImpl.java

```diff
@@ -25587,7 +25610,7 @@ public Mono<Response<Void>> rebootNodeInternalWithResponseAsync(String poolId, S
             }
         });
         return FluxUtil.withContext(context -> service.rebootNodeInternal(this.getEndpoint(),
-            this.getServiceVersion().getVersion(), contentType, poolId, nodeId, accept, requestOptionsLocal, context));
+            this.getServiceVersion().getVersion(), poolId, nodeId, accept, requestOptionsLocal, context));
```

Comment:
mark, the default setted content-type is removed. And it should be passed from client method.